container/docker/docker.go

```diff
@@ -132,6 +133,10 @@ func Version() ([]int, error) {
 	return parseDockerVersion(VersionString())
 }
 
+func APIVersion() ([]int, error) {
+	return parseDockerAPIVersion(APIVersionString())
+}
+
 func VersionString() string {
```

Comment:
Not at all. There are many places this function used and I don't want them to disturb.